src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -39,24 +39,22 @@
 
     @Override
     protected void prepareTestData() {
-        newSession = new FeedbackSessionAttributes();
-        newSession.setCourseId("CFeedbackUiT.CS1101");
-        newSession.setFeedbackSessionName("New Session ##");
-        // start time is in future, hence the year.
-        newSession.setStartTime(TimeHelper.convertToDate("2035-04-01 11:59 PM UTC"));
-        newSession.setEndTime(TimeHelper.convertToDate("2035-04-30 10:00 PM UTC"));
-        newSession.setCreatorEmail("teammates.test1@gmail.tmt");
-        newSession.setCreatedTime(Const.TIME_REPRESENTS_NEVER);
-        newSession.setSessionVisibleFromTime(Const.TIME_REPRESENTS_FOLLOW_OPENING);
-        newSession.setResultsVisibleFromTime(Const.TIME_REPRESENTS_LATER);
-        newSession.setGracePeriod(0);
-        newSession.setInstructions(new Text("Please fill in the new feedback session."));
-        newSession.setSentOpenEmail(false);
-        newSession.setSentPublishedEmail(false);
-        newSession.setTimeZone(8.0);
-        newSession.setFeedbackSessionType(FeedbackSessionType.STANDARD);
-        newSession.setClosingEmailEnabled(true);
-        newSession.setPublishedEmailEnabled(true);
+        newSession = FeedbackSessionAttributes
+                .builder("New Session ##", "CFeedbackUiT.CS1101", "teammates.test1@gmail.tmt")
+        .withStartTime(TimeHelper.convertToDate("2035-04-01 11:59 PM UTC"))
```

Comment:
I don't usually pick on indentations, but shouldn't all these `with*`s and `build()` be indented 8 spaces further to the right?